core/tls.cabal

```diff
@@ -53,8 +53,9 @@ Library
                    , asn1-types >= 0.2.0
                    , asn1-encoding
                    , x509 >= 1.7.5
-                   , x509-store >= 1.6
+                   , x509-store >= 1.6.8
```

Comment:
See https://github.com/vincenthz/hs-certificate/pull/117

Reply:
This is a big issue since Vincent stopped using Haskell. Somebody should take over x509-store.